components/event-publisher-proxy/pkg/sender/nats.go

```diff
@@ -18,48 +18,45 @@ type GenericSender interface {
 	Send(context.Context, *cev2event.Event) (int, error)
 }
 
-// BebMessageSender is responsible for sending messages over HTTP.
+// NatsMessageSender is responsible for sending messages over HTTP.
 type NatsMessageSender struct {
 	ctx               context.Context
 	logger            *logrus.Logger
-	backendConnection *pkgnats.BackendConnection
+	backendConnection *pkgnats.Connection
 }
 
-// NewNatsMessageSender returns a new NewNatsMessageSender instance with the given nats connection.
-func NewNatsMessageSender(ctx context.Context, bc *pkgnats.BackendConnection, logger *logrus.Logger) *NatsMessageSender {
+// NewNatsMessageSender returns a new NewNatsMessageSender instance with the given NATS connection.
+func NewNatsMessageSender(ctx context.Context, bc *pkgnats.Connection, logger *logrus.Logger) *NatsMessageSender {
```

Comment:
would it be possible to rename `pkgnats` to `nats` or something similar

Reply:
In theory yes. We do import 2 packages in this file which are called `nats`.
Both are using a custom import name. 

```go
	cenats "github.com/cloudevents/sdk-go/protocol/nats/v2"
	pkgnats "github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/nats"
```

I would agree renaming it if we would not have a collision on the import names. However I am not sure why we should rename to `pkgnats` and not `cenats` to `nats` then. I like the current version where both imports use an explicit name.

What do you think ?